example-package-no-core/elm.json

```diff
@@ -1,8 +1,12 @@
 {
   "type": "package",
-  "name": "example ",
+  "name": "elm-explorations/node-test-runner-example-package",
   "summary": "SSCCE on replicating issue 273",
+  "license": "BSD-3-Clause",
   "version": "4.0.1",
+  "exposed-modules": [
+    "Something"
+  ],
```

Comment:
This package elm.json was broken so it didn’t test what it was supposed to.

Reply:
Question is what is it supposed to test? 

This is the issue I fixed and added this as a regression test for: https://github.com/rtfeldman/node-test-runner/issues/273
@Maxim-Filimonov put together a SSCCE [here](https://github.com/Maxim-Filimonov/node-test-runner-273-sscce) but looking at that SSCCE the elm.json file has no dependancy on elm/core so it should fail! @Maxim-Filimonov I realise it was over a year ago that you made the SSCCE but can you remember why you expected a package with no dependancy on elm/core to pass testing?

Reply:
I am thinking we can delete this example.

Reply:
Removed in https://github.com/rtfeldman/node-test-runner/pull/448